pyomo/contrib/incidence_analysis/interface.py

```diff
@@ -16,12 +16,17 @@
 from pyomo.core.base.reference import Reference
 from pyomo.core.expr.visitor import identify_variables
 from pyomo.common.collections import ComponentSet, ComponentMap
-from pyomo.common.dependencies import scipy_available
+from pyomo.common.dependencies import scipy_available, networkx_available
 from pyomo.contrib.incidence_analysis.matching import maximum_matching
 from pyomo.contrib.incidence_analysis.triangularize import block_triangularize
+from pyomo.contrib.incidence_analysis.dulmage_mendelsohn import (
+        dulmage_mendelsohn,
+        )
 if scipy_available:
     from pyomo.contrib.pynumero.interfaces.pyomo_nlp import PyomoNLP
     import scipy as sp
+if networkx_available:
+    import networkx as nx
```

Comment:
You can just:
```python
from pyomo.common.dependencies import networkx as nx
```
This has the side benefit of raising an error about an unavailable optional dependency in `get_incidence_graph` instead of `NameError: name 'nx' is not defined`